### DISCUSSION TITLE
A question about matching lat-lon grids for the compset F2010-CICE

### DISCUSSION THREAD

**Discussion by dongxthu** [E3SM-Project/E3SM#6782] [E3SM model help]
Dear Team,

I’m looking for a lat-lon grid that can be matched to the air-land coupled compset "F2010-CICE", because I need to couple it with other tools. However, I encountered errors when attempting to use classic lat-lon grids such as f09_g16 and f19_g16 when doing the setup. Errors are like: "CAM build-namelist - ERROR: No default value found for ncdata" or "ERROR: CLM build-namelist::ELMBuildNamelist::add_default() : No default value found for fsurdat".

Based on my investigation, this error might be due to E3SM not supporting such combinations. I wonder if there are any lat-lon grids (only the land model using lat-lon grid is also OK) that can match this compset in E3SM v3?

Thank you for your time and assistance!!

Best,
Don

**Comment by zhangshixuan1987** [comment]
@dongxthu : the current E3SM used MPAS sea ice, the CICE is not supported anymore as far as I know.

**Reply by dongxthu** [reply]
Thank you so much! I will try other grids and methods.

**Comment by rljacob** [comment]
Its true we don't support CICE.  Also we do not support regular lat-lon grids in the atmosphere model.   The old FV dycore that uses those is still there but, as you've learned, the machinery to set up a case is broken.

**Reply by dongxthu** [reply]
Thank you so much! I will try other grids and methods.
